src/components/Vega.svelte

```diff
@@ -81,6 +81,8 @@
   $: updateSpec(patchedSpec);
   $: updateSignals(vegaPromise, signals);
 
+  export const vegaAccessor = () => vegaPromise.then((v) => v.view); // should be exported.
```

Comment:
I added the comment because I wasn't seeing "export" in the web page (due to compiler?), and not seeing changes at all, which turned out to be due to brokenness in the Emacs mode(s) so that changes were not being saved.   Deleted this comment in subsequent commit.